### PR TITLE
ASoC:SOF:skl add cldma copy firmware

### PR DESCRIPTION
**Pull request by zhuyingjiang**
Signed-off-by: Zhu Yingjiang <yingjiang.zhu@linux.intel.com>

### CODE REVIEW

**Comment by zhuyingjiang** [comment]
@lgirdwood

**Comment by lgirdwood** [comment]
@zhuyingjiang what does this uodate do, ca you describe in the commit messages.

File: sound/soc/sof/intel/hda-loader-skl.c [hunk 1]
```diff
@@ -418,3 +445,140 @@ int hda_dsp_cl_boot_firmware_skl(struct snd_sof_dev *sdev)
...
+			cl_skl_cldma_fill_buffer(sdev, bufsize, bytes_left,
+						 curr_pos, false, true);
+			return 0;
+		}
+	}
+
+	return bytes_left;
+}
+
+#define STRIP_FIRMWARE		1
```
Comment (lgirdwood) [inline sound/soc/sof/intel/hda-loader-skl.c]: Whats this for, I don't think we need it for SOF ?

Comment (zhuyingjiang) [reply sound/soc/sof/intel/hda-loader-skl.c]: This is for the offset of the binary, when using cAVS binary for debug, we need set it as true, and SOF don't need this. I will remove it.

**Comment by zhuyingjiang** [comment]
@lgirdwood commit message added, and the STRIP_FIRMWARE #define is removed.

**Pull request merged**